src/main/resources/tags/info/floodgatemismatch.tag

```diff
@@ -0,0 +1,7 @@
+type: text
```

Comment:
Might want to mention there that this is not the issue if all Floodgate instances on a particular network are updated to 2.1.0-SNAPSHOT.

Reply:
Does this fix work?